man1/aur-repo.1

```diff
@@ -47,13 +47,28 @@ when checking for updates. Implies
 .TP
 .BR \-l ", " \-\-list
 List the contents of the repository in
-.BI pkgname \et pkgver
+.BI pkgname \et pkgver \en
 format. If the
 .B \-S
 option is not enabled, the repository is queried directly through the
 .BI .db
 file on-disk.
 
+.TP
+.B \-\-table
+Change the format used by
+.B \-\-list
+from
+.BI pkgname \et pkgver \en
+to
+.BI pkgname \et depends \et pkgbase \et pkgver \en\c
+\&. Missing values are represented by
```

Comment:
github only allows onlined suggestions... :disappointed:

Doc update to go with the above change:
```suggestion
\&. If a package has no dependencies, \fBpkgname\fR is used in its place, otherwise missing values are represented by
```